build-support/bin/isort.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import argparse
+import re
+import subprocess
+from typing import List, Iterable
+
+from common import die
+
+
+def main() -> None:
+  args = create_parser().parse_args()
+  result = run_isort(fix=args.fix)
+  parse_result(result)
+
+
+def create_parser() -> argparse.ArgumentParser:
+  parser = argparse.ArgumentParser(description="Run isort over build-support to ensure valid import order.")
```

Comment:
Erm - more disease here of not just using a task plugin in the source trees we already lint. Having to setup argparse, etc.